packages/backend/src/submissions/submissions.controller.ts

```diff
@@ -30,12 +32,12 @@ export class SubmissionsController {
   }
 
   @Patch('/approve/:id')
-  approve(@Param('id:', ParseUUIDPipe) id: string) {
+  approve(@Param('id:', ParseUUIDPipe) id: string): Promise<Link> {
     return this.submissionsService.approve(id)
   }
 
   @Patch('/decline/:id')
-  decline(@Param('id:', ParseUUIDPipe) id: string) {
+  decline(@Param('id:', ParseUUIDPipe) id: string): Promise<SubmissionEntitiy> {
     return this.submissionsService.decline(id)
   }
```

Comment:
remove the `:` from `@Param('id:'`, it can't extract the id from the request like this.

Also, the routes should be `/:id/approve` and `/:id/decline`, because there are other routes where the first part is parsed as a variable. Sending a request with the url `/approve/...` could end up in another controller method where the value of the `id` is `approve`